### ISSUE TITLE
[e2e] Create a minimal set of tests to ensure that our a11y rating does not regress

### ISSUE THREAD

**Issue by unlikelyzero** [nasa/openmct#6389]
#### Summary
We are working towards Section 508 / WCAG2.0AA Compliance for Build 9. We will be able to get an exhaustive report and start addressing the failures individually. After this assessment is completed, the axe tool can be integrated with playwright to start preventing regressions.

#### Approach
There are currently two separate packages available `axe-playwright` and `@axe-core/playwright` which are roughly the same in terms of rules engine but very different in implementation, workflow, and reporting.

https://playwright.dev/docs/accessibility-testing#example-accessibility-tests
https://github.com/abhinaba-ghosh/axe-playwright#readme

A WIP Branch is available here https://github.com/nasa/openmct/tree/playwright-axe
